quickstarts/minikube/index.md

```diff
@@ -0,0 +1,55 @@
+# Starting Minikube 
+
+This assumes that you have the latest version of the `minikube` binary, which you can get [here](https://kubernetes.io/docs/setup/minikube/#installation).
+
+```shell
+minikube start ...
+```
+
+> NOTE: Make sure to start `minikube` with your configured VM. If need help look at the [documentation](https://kubernetes.io/docs/setup/minikube/#quickstart) for more.
+
+Once Minikube this is started, let's create our `kafka` namespace:
+
+```shell
+kubectl create namespace kafka 
+```
+
+# Applying Strimzi installation file
+
+Next we apply the Strimzi install files, including `ClusterRoles`, `ClusterRoleBindings` and some **Custom Resource Definitions** (`CRDs`). The CRDs define the schemas used for declarative management of the Kafka cluster, Kafka topics and users.
+
+```shell
+curl -L https://github.com/strimzi/strimzi-kafka-operator/releases/download/0.8.2/strimzi-cluster-operator-0.8.2.yaml \
```

Comment:
We need to update this link for every release? :-(

Reply:
I could do a VARIABLE ?